src/common/locations.ts

```diff
@@ -110,6 +110,8 @@ export function getLocationNameForFips(fips: string): string {
 }
 
 export function getLocationUrlForFips(fips: string): string {
+  // TODO(pablo): Make sure that these urls are uniform, here, the state
+  // URL ends on / but not the county one
```

Comment:
Hrm.  Why should the state end with / but not the county one?

Reply:
We probably missed that when implementing the function. I'm going to check where this function is used and update it so it's consistent once we ship Explore.